src/components/NavigationAccount.vue

```diff
@@ -106,16 +99,15 @@
 		<template #extra>
```

Comment:
NcAppNavigationCaption doesnt have an extra slot, need to find a way how to make account settings work

Reply:
@marcoambrosini should i add an extra slot at appnavigataioncaption, or should we use a different component?

Reply:
Maybe use a <Fragment> component @GretaD https://www.npmjs.com/package/vue-fragment
cc @ShGKme @susnux for opinions

Reply:
Yes, fragment will work here for Vue 2. But we are usually using `vue-frag`: https://www.npmjs.com/package/vue-frag 

Mail already has it installed.